### PR TITLE
[CELEBORN-769] Change default value of celeborn.client.push.maxReqsInFlight to 16

### PR DESCRIPTION
**Pull request by waitinfuture**
…Flight to 16

### What changes were proposed in this pull request?
Change default value of celeborn.client.push.maxReqsInFlight to 16.

### Why are the changes needed?
Previous value 4 is too small, 16 is more reasonable.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Pass GA.

### CODE REVIEW

**Comment by waitinfuture** [comment]
cc @AngersZhuuuu @pan3793

File: docs/configuration/client.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ license: |
 | celeborn.client.push.limit.inFlight.sleepInterval | 50ms | Sleep interval when check netty in-flight requests to be done. | 0.3.0 | 
 | celeborn.client.push.limit.inFlight.timeout | &lt;undefined&gt; | Timeout for netty in-flight requests to be done.Default value should be `celeborn.client.push.timeout * 2`. | 0.3.0 | 
 | celeborn.client.push.limit.strategy | SIMPLE | The strategy used to control the push speed. Valid strategies are SIMPLE and SLOWSTART. The SLOWSTART strategy usually works with congestion control mechanism on the worker side. | 0.3.0 | 
-| celeborn.client.push.maxReqsInFlight | 4 | Amount of Netty in-flight requests per worker. The maximum memory is `celeborn.client.push.maxReqsInFlight` * `celeborn.push.buffer.max.size` * compression ratio(1 in worst case), default: 64KiB * 4 = 256KiB | 0.3.0 | 
+| celeborn.client.push.maxReqsInFlight | 16 | Amount of Netty in-flight requests per worker. The maximum memory is `celeborn.client.push.maxReqsInFlight` * `celeborn.push.buffer.max.size` * compression ratio(1 in worst case), default: 64KiB * 4 = 256KiB | 0.3.0 |
```
Comment (pan3793) [inline docs/configuration/client.md:46]: the default values calculation should be updated all well?

Comment (waitinfuture) [reply docs/configuration/client.md]: > the default values calculation should be updated all well?

Updated, thanks, please take a look again.

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 2]
```diff
@@ -2675,9 +2675,10 @@ object CelebornConf extends Logging {
       .version("0.3.0")
       .doc("Amount of Netty in-flight requests per worker. The maximum memory is " +
         "`celeborn.client.push.maxReqsInFlight` * `celeborn.push.buffer.max.size` * " +
-        "compression ratio(1 in worst case), default: 64KiB * 4 = 256KiB")
+        "number of workers * compression ratio(1 in worst case), say we have 50 workers," +
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:2678]: nit: a space at the end of line

**Review by pan3793 — APPROVED** [review]

**Comment by cfmcgrady** [comment]
changing the key from `celeborn.client.push.maxReqsInFlight` to `celeborn.client.push.maxRequestsInFlight`? it's better to avoid using abbreviations in configurations. WDYT? @pan3793 @waitinfuture

**Comment by pan3793** [comment]
@cfmcgrady We use `Reqs` in several configurations, and "max" also is an abbr of "maximum" :)

**Comment by cfmcgrady** [comment]
> @cfmcgrady We use `Reqs` in several configurations, and "max" also is an abbr of "maximum" :)

ok, Spark also has the key like `spark.reducer.maxReqsInFlight`

**Comment by waitinfuture** [comment]
> changing the key from `celeborn.client.push.maxReqsInFlight` to `celeborn.client.push.maxRequestsInFlight`? it's better to avoid using abbreviations in configurations. WDYT? @pan3793 @waitinfuture

+1 personally, but I think it's OK to use abbreviations, sometimes whole word is too long 😄

**Pull request closed without merging**